aliases.go

```diff
@@ -5,8 +5,6 @@ import (
 	"strings"
 )
 
-// #region Create
-// TODO: make sure duplicate aliases throw an error that we can catch
```

Comment:
Removed TODO because 1. we have a function call to remove duplicates and 2. it's not possible to make a "duplicate alias" anyways, so returning an error for that is unnecessary.